vars/commonPipelineEnvironment.groovy

```diff
@@ -187,11 +187,23 @@ class commonPipelineEnvironment implements Serializable {
         [filename: '.pipeline/commonPipelineEnvironment/git/commitId', property: 'gitCommitId'],
         [filename: '.pipeline/commonPipelineEnvironment/git/commitMessage', property: 'gitCommitMessage'],
         [filename: '.pipeline/commonPipelineEnvironment/mtarFilePath', property: 'mtarFilePath'],
+        [filename: '.pipeline/commonPipelineEnvironment/configurationFile', property: 'configurationFile'],
         [filename: '.pipeline/commonPipelineEnvironment/abap/repositoryNames', property: 'abapRepositoryNames'],
     ]
 
     void writeToDisk(script) {
 
+        // The default config files were extracted from merged library resources
+        // by setupCommonPipelineEnvironment.groovy into folder .pipeline/tmp/defaults/.
```

Comment:
I think the comment is garbled (copy/paste from earlier location). It doesn't really describe what happens.